ecl/hql/hqlcache.cpp

```diff
@@ -342,7 +342,7 @@ void convertSelectsToPath(StringBuffer & filename, const char * eclPath)
 
 //---------------------------------------------------------------------------------------------------------------------
 
-static IHqlExpression * createSimplifiedDefinition(ITypeInfo * type)
+static IHqlExpression * createSimplifiedDefinitionType(ITypeInfo * type, bool implicitConstantType=false)
```

Comment:
not sure I like the name change - it implies it returns a type.  Possibly createSimplifiedDefinitionFromType()